src/views/Claim/Claim.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Container, Spacer } from "react-neu";
+
+import Page from "components/Page";
+import PageHeader from "components/PageHeader";
+import Split from "components/Split";
+
+import ClaimCard from "./components/ClaimCard";
+import VestingNotice from "./components/VestingNotice";
+
+const Claim: React.FC = () => {
+  return (
+    <Page>
+      <PageHeader icon="🦋" subtitle="" title="Claim YAMs!" />
+      <Container>
+        <Spacer />
+        <VestingNotice />
```

Comment:
no need to add `<VestingNotice />` on the claim page, it was only for the migration page